test/specs/rules/validate-indentation.js

```diff
@@ -156,6 +156,23 @@ describe('rules/validate-indentation', function() {
         ]);
     });
 
+    it('should not report errors for using tabs when validateIndentation is a number', function() {
+        checker.configure({ validateIndentation: 4 });
```

Comment:
@kylepaulsen if you work on different project (one that can be pulled from github) it should contains it's own .jscsrc config with `validateIndentation` rules, right?